### ISSUE TITLE
NSP output not being shown by Hawkeye scan

### ISSUE THREAD

**Issue by JoeWroe** [Stono/hawkeye#57]
### EXPECTED BEHAVIOUR

Running `$ hawkeye scan` returns an issue from NSP when it is possible to run `$ nsp check` and get an issue output.

### ACTUAL BEHAVIOUR

running `$ nsp check` returns the following output:

```
$ nsp check

(+) 1 vulnerability found
```

with a pretty little table at the bottom.

but running `$ hawkeye scan` seems to silence the error:

```
$ hawkeye scan

…

[info] Running module Node Security Project
[info]  -> /…/node_modules/nsp/bin/nsp check -o json
[info] Not Handling Bandit Scan
[info] Not Handling Python Outdated Dependencies Scan
[info] Not Handling Safety Scan
[info] Not Handling Brakeman Scan
[info] Not Handling Bundler Scan
[info] scan complete, 0 issues found .         <~~~~~ [Unexpected output]

[info] Doing writer: console
[info] Scan complete
```

### ADDITIONAL INFO

1. We have NSP installed globally.

**Comment by nicolastrres** [comment]
@JoeWroe could you try the latest version (1.0.4)? It was fixed on #59 so it should be working on the latest version

**Comment by felixhammerl** [comment]
This will be obsoleted by #66, as `nsp` will be removed from Hawkeye.
